tensorflow/lite/micro/tools/make/Makefile

```diff
@@ -97,7 +97,7 @@ MICROLITE_LIB_NAME := libtensorflow-microlite.a
 # These two must be defined before we include the target specific Makefile.inc
 # because we filter out the examples that are not supported for those targets.
 # See targets/xtensa_xpg_makefile.inc for an example.
-MICRO_LITE_EXAMPLE_TESTS := $(shell find tensorflow/lite/micro/examples/ -name Makefile.inc)
+MICRO_LITE_EXAMPLE_TESTS := $(shell find tensorflow/lite/micro/examples/ -maxdepth 2 -name Makefile.inc)
```

Comment:
could you describe what the reason for having maxdepth=2 is? what issue were you running into without this?

Reply:
Without limiting maxdepth to 2, this command is finding all Makefile.inc files from examples and its target specific Makefiles, that are also included in example Makefile.inc. This makes that target specific Makefile from example is included twice, e.g. in micro_speech example:

https://github.com/tensorflow/tensorflow/blob/3448ea1f3d321d9d3d0487b1680a75a5c7ce05bd/tensorflow/lite/micro/examples/micro_speech/Makefile.inc#L238